apps/openmw/mwmechanics/spellcasting.hpp

```diff
@@ -21,6 +22,8 @@ namespace MWMechanics
 
     ESM::Skill::SkillEnum spellSchoolToSkill(int school);
 
+    float calcEffectCost(ESM::ENAMstruct effect);
```

Comment:
non-primitive types should always be passed by const reference. If you don't, cppcheck will complain (and the resulting code might be slower, though maybe some compilers will optimize it?)